test/models/download_request_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class DownloadRequestTest < ActiveSupport::TestCase
+
+  before(:all) do
+    @azul_hash = {
+        'Fake.HCA.Study.1' => [
+          {
+            'source' => 'hca',
+            'count' => 1,
+            'upload_file_size' => 10.megabytes,
+            'file_format' => 'loom',
+            'file_type' => 'analysis_file',
+            'accession' => 'FakeHCAStudy1',
+            'project_id' => 'hca_project_id'
+          }
+        ],
+        'Fake.HcaS.tu.d.y.2' => []
+      }
+      
+    @encoded_azul_hash = "{\"Fake.HCA.Study.1\":[{\"source\":\"hca\",\"count\":1,\"upload_file_size\":10485760,\"file_format\":\"loom\",\"file_type\":\"analysis_file\",\"accession\":\"FakeHCAStudy1\",\"project_id\":\"hca_project_id\"}],\"Fake.HcaS.tu.d.y.2\":[]}"
+
+  end
+
```

Comment:
BLOCKER: This test will be much more robust if it confirms a DownloadRequest can be saved to the database, and then read back

e.g.
download_req = DownloadRequest.new({ ..., azul_files: { <<hash >>}})
download_req.save!
assert_equal encoded_azul_hash, download_req.azul_files_as_hash

Reply:
(and yup, I realize ClusterFileInfoTest doesn't have such a test -- that was an oversight by me :P )